ref/general/basic-registry.adoc

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: Configuring a basic user registry
+:seo-description: Open Liberty provides an easy-to-use basic user registry for developers.
+:page-layout: general-reference
+:page-type: general
+= Configuring a basic user registry
+
+A user registry is a database that contains account information, such as usernames and passwords, that can be retrieved by applications to authenticate and authorize users and services.
+
+Typically, microservice-based applications in production run on an application server that is configured to use an external registry, such as a Lightweight Directory Access Protocol (LDAP) registry. Applications access information in the registry by using APIs, such as link:docs/ref/javaee/8/#package=javax/annotation/security/package-frame.html&class=overview-summary.html[the Jakarta EE Security API]. However, in test environments, a developer might want to set up a basic, file-based user registry within the application server to simplify security operations during the development process. Open Liberty provides an easy-to-use basic user registry for developers.
+
+== The Application Security feature
+
+The `appSecurity` feature provides the core capability for the Java EE Security API. This feature must be enabled in the `server.xml` file in order to configure a basic user registry. For more information, see link:/docs/ref/feature/#appSecurity-3.0.html[the Application Security feature].
+
+== Basic user registry for test environments
+
+Instead of being hosted externally, Open Liberty's basic user registry is configured directly in the `server.xml` file, or in an adjacent xml file on the server. Although this configuration is not secure enough for production environments, it can help simplify authentication and authorization scenarios during development and testing. The basic user registry provides a fast and easy way to test the access to your protected application.
```

Comment:
Should `xml` in "...or in an adjacent xml file on the server." be all caps (`XML`)?

Reply:
yes- thanks good catch